acceptance/Rakefile

```diff
@@ -96,7 +91,8 @@ EOS
 
   overriding_options = ENV['OPTIONS']
 
-  args = ["--options-file", options_file, config_opt, tests_opt, overriding_options].compact
+  args = ["--options-file", options_file, config_opt, tests_opt,
```

Comment:
This causes an error when I try to run

    Jamess-MacBook-Pro:acceptance jstocks$ rake ci:test:aio
    Adding Redhat 7 master to ENV['TEST_TARGET']. Value is now 'redhat7-64m-redhat7-64a'
    ./config/aio/options.rb
    ./local_options.rb
    rake aborted!
    NoMethodError: undefined method `split' for nil:NilClass
    /Users/jstocks/GitHub/cthun-agent/acceptance/Rakefile:95:in `beaker_test'
    /Users/jstocks/GitHub/cthun-agent/acceptance/Rakefile:276:in `block (3 levels) in <top (required)>'
    /Users/jstocks/.rvm/gems/ruby-2.1.1/bin/ruby_executable_hooks:15:in `eval'
    /Users/jstocks/.rvm/gems/ruby-2.1.1/bin/ruby_executable_hooks:15:in `<main>'
    Tasks: TOP => ci:test:aio
    (See full trace by running task with --trace)

Is this because the new code does not handle ENV['OPTIONS'] being unset?